src/main/java/com/mattermost/integration/figma/input/file/notification/User.java

```diff
@@ -0,0 +1,13 @@
+package com.mattermost.integration.figma.input.file.notification;
+
+import lombok.*;
+
+@Getter
+@Setter
```

Comment:
pls take a look on lombok "@ Data" annotation I believe it aggregates most of this annotations